docs/requirements/requirements.md

```diff
@@ -0,0 +1,112 @@
+# Hardware requirements of TwPM module
+
+[tcg_client_v13r27]:<https://trustedcomputinggroup.org/wp-content/uploads/TCG_PCClientTPMInterfaceSpecification_TIS__1-3_27_03212013.pdf>
+[tcg_client_PTP_v1p5r14]:<https://trustedcomputinggroup.org/wp-content/uploads/PC-Client-Specific-Platform-TPM-Profile-for-TPM-2p0-v1p05p_r14_pub.pdf>
+
+## Acronyms
+
+Legend:
+
+* PTP – Platform TPM Profile
+* CRB – Command Response Buffer interface
+* DDWG – Device Driver’s Writers Guide
+* Certification PP – Certification Protection Profile
+* TIS – TPM Interface Specification
+
+## Requirements
+
+### Power supply
+
+#### Power supply voltage
+
+* TPM MUST support a supply voltage of 1.8V or 3.3V.
+* TPM MAY support supply and I/O voltages of both 1.8V and 3.3V.
+* TPM MAY support other supply voltages.
+
+Source: [tcg_client_v13r27] l.1373-1375
+
+#### Power supply max current
+
+Maximum power supply taken by TPM MUST be 250mA.
+
+Source: [tcg_client_PTP_v1p5r14], Table 60, VDD row
+
+### SPI Interface
+
+#### Frequency
+
+Default clock frequency for PC Client platforms is defined to be 24MHz.
+Clock frequency signal, minimum, is 10MHz.
```

Comment:
TPM must work properly on frequencies from 10MHz to 24MHz, but even now spec mentions that this may change in the future. We could mention that hardware which supports higher frequencies is very welcome.

Reply:
(corrected, added line 40 in https://github.com/Dasharo/twpm-docs/pull/4/commits/de6f283a8ec1c163518dbe4be392c4cb8e29d841)

Reply:
Document doesn't tell why we recommended platforms wight higher frequencies, I suggest something like:
```
TPM must be capable of operating at frequencies 10 MHz - 24 MHz. However, the
maximum frequency requirement may be raised in the future, so the target
platform should have SPI capable of higher frequencies.
```

Reply:
@arturkow2000 corrected, in current commit.